docs/app/Examples/modules/Dimmer/Variations/DimmerExampleBlurring.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import { Button, Dimmer, Segment } from 'semantic-ui-react'
+
+export default class DimmerExampleBlurring extends Component {
+  state = {}
+
+  handleShow = () => this.setState({ active: true })
+  handleHide = () => this.setState({ active: false })
+
+  render() {
+    const { active } = this.state
+
+    return (
+      <div>
+        <Dimmer blurring dimmable={Segment} dimmed={active}>
```

Comment:
This currently does not pass extra props to the Segment.  Example, adding `color='red'` should pass `color` to the `Segment`.  What if we went with this API, it does pass extra props:

```jsx
<Dimmer.Dimmable as={Segment} blurring dimmed={active} color='red'>
  <Dimmer active={active} />

  <p>
    <img src='http://semantic-ui.com/images/wireframe/short-paragraph.png' />
  </p>
  <p>
    <img src='http://semantic-ui.com/images/wireframe/short-paragraph.png' />
  </p>
</Dimmer.Dimmable>
```

I tested the above with only these changes required:

1. Expose `Dimmer.Dimmable = DimmerDimmable`
1. Rename `DimmmerDimmable` prop `Component` to `as` and use our standard `getElementType()` util and render `<ElementType />` instead.
1. I did not do this, but we should then remove all `dimmable` prop reference and logic from the main `Dimmer`.

This might be more in line with our existing patterns.